components/globalVariables/global-variable-overview.tsx

```
@@ -37,36 +42,216 @@ export const GlobalVariableOverview: FC<GlobalVariableOverviewProps> = ({
+            </div>
+          </div>
+        )}
+
+        {/* Additional Information */}
+        {globalVariable.trackingInstructions && (
+          <div className="neobrutalist-container md:col-span-2">
+            <h2 className="neobrutalist-h2">Tracking Instructions</h2>
+            <div
+              className="prose max-w-none"
+              dangerouslySetInnerHTML={{
+                __html: globalVariable.trackingInstructions,
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)